crates/edr_napi/src/debug_trace.rs

```diff
@@ -36,3 +36,54 @@ pub struct DebugTraceLogItem {
     /// Map of all stored values with keys and values encoded as hex strings.
     pub storage: Option<HashMap<String, String>>,
 }
+
+/// Result of `debug_traceTransaction` and `debug_traceCall` after
+/// normalisation. `pass` and `gas_used` exist before normalisation. They will
+/// be replaced by `failed` and `gas` respectively. They currently exist
+/// together because Hardhat still depends on them but `pass` and `gas_used`
+/// should likely to be removed after a while.
+#[napi(object)]
+pub struct RpcDebugTraceResult {
```

Comment:
As this type is no longer part of the API, do we really need to expose it?

I assume Hardhat has its own definition, given that it's a standard.

cc: @fvictorio

Reply:
You are right, Hardhat has its own definition